### PR TITLE
change bingmaps key, added ac-map-layer-provider

### PR DESCRIPTION
**Pull request by katzi0**

### CODE REVIEW

File: packages/client/src/app/game/views/game-map/game-map.component.ts [hunk 1]
```diff
@@ -65,6 +66,8 @@ export class GameMapComponent implements OnInit, OnDestroy {
       if (!this.createPathMode) {
         this.viewerOptions.setFpvCameraOptions(viewer);
       }
+      this.mapLayerProviderOptions = MapLayerProviderOptions.BingMaps;
+
```
Comment (yuvalbl) [inline packages/client/src/app/game/views/game-map/game-map.component.ts]: remove space

File: packages/client/src/app/game/views/game-map/game-map.component.html [hunk 2]
```diff
@@ -1,4 +1,11 @@
 <ac-map>
+  <ac-map-layer-provider
+    [provider]="mapLayerProviderOptions"
+    [options]="{
+                                   url : 'https://dev.virtualearth.net',
```
Comment (yuvalbl) [inline packages/client/src/app/game/views/game-map/game-map.component.html]: move object to ts file

File: packages/client/src/app/game/views/game-container/game-container.component.ts [hunk 3]
```diff
@@ -47,6 +47,8 @@ export class GameContainerComponent implements OnInit, OnDestroy {
               private ngZone: NgZone,
               public controlledService: TakeControlService,
               private snackBar: MatSnackBar) {
+    Cesium.BingMapsApi.defaultKey = 'AkXEfZI-hKtZ995XgjM0XHxTiXpyS4i2Vb4w08Pjozwn-NAfVIvvHBYaP6Pgi717';
```
Comment (yuvalbl) [inline packages/client/src/app/game/views/game-container/game-container.component.ts]: remove empty line + format

**Comment by katzi0** [comment]
map-provider component containing ac-map-layer-provider (angular-cesium component)

**Pull request merged**